src/SFA.DAS.EmployerIncentives.Api/NLogConfiguration.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.ApplicationInsights.NLogTarget;
+using NLog;
+using NLog.Common;
+using NLog.Config;
+using NLog.Targets;
+using SFA.DAS.NLog.Targets.Redis.DotNetCore;
+using System;
+using System.IO;
+
+namespace SFA.DAS.EmployerIncentives.Api
+{
+    public class NLogConfiguration
+    {
+        public void ConfigureNLog()
```

Comment:
Devops prefer us to use nlog.config,  Please update to use that method for configuring nlog.

Reply:
confirmed they no longer do :)